python/mxnet/symbol/numpy/random.py

```diff
@@ -86,6 +86,7 @@ def uniform(low=0.0, high=1.0, size=None, **kwargs):
     Notes
     -----
     This function currently does not support ``low`` and ``high`` as symbols.
+    `size` is not optional.
```

Comment:
I'd rather remove the `, optional` part in the `Parameter` section above for `size`.

Reply:
fixed and updated notes